gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.10-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.10.2-bin.zip
```

Comment:
That's a bit nasty to hide that in here 😉

Reply:
Not sure what you mean? Is that a big change?

Reply:
No, but it's totally unrelated to the commit title of "Remove DefaultSelfResolvingDependency", isn't it?

Reply:
Oh yea 100% sorry about that, that repo usually doesn't get much review, I'll make sure to make cleaner commits moving on!